package.json

```diff
@@ -20,5 +17,8 @@
   "bugs": {
     "url": "https://github.com/ahdinosaur/prebuildify-cross/issues"
   },
-  "homepage": "https://github.com/ahdinosaur/prebuildify-cross#readme"
+  "homepage": "https://github.com/ahdinosaur/prebuildify-cross#readme",
+  "peerDependencies": {
+    "prebuildify": "*"
```

Comment:
Is this temporary or permanent? I hate peerDependencies :)

Reply:
Wut. Where the hell did this come from?

Reply:
Oh nevermind, I thought this was in `leveldown` repo lol. Yes, I don't like it either.